.github/workflows/migrator.yml

```diff
@@ -12,62 +12,143 @@ on:
         required: true
         type: number
 
+permissions:
+  contents: write
+  pull-requests: read
+
 jobs:
-  recreate:
+  migrate:
     if: |
        (
         github.event.issue.pull_request != null &&
-        contains(github.event.comment.body, '@pyansys-ci-bot migrate') &&
-        github.event.comment.user.login == 'germa89' 
+        (contains(github.event.comment.body, '@pyansys-ci-bot migrate') || contains(github.event.comment.body, '@pyansys-ci-bot sync') )&&
+        github.event.comment.user.login == 'germa89'
```

Comment:
Only you allowed? I would avoid it -- better move to using a team (``pymapdl-maintainers``)

Reply:
Done!